<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shivani" failures="1" tests="5" name="Test" time="45.117" errors="0" timestamp="2023-06-22T23:29:34 IST">
  <testcase classname="createContactTests.createContactTest" name="createContTest" time="6.008">
    <skipped/>
  </testcase> <!-- createContTest -->
  <testcase classname="createContactTests.createContactTest" name="createContTest" time="5.845">
    <skipped/>
  </testcase> <!-- createContTest -->
  <testcase classname="createContactTests.createContactTest" name="createContTest" time="5.761">
    <skipped/>
  </testcase> <!-- createContTest -->
  <testcase classname="createContactTests.createContactTest" name="createContTest" time="5.565">
    <skipped/>
  </testcase> <!-- createContTest -->
  <testcase classname="createContactTests.createContactTest" name="createContTest" time="5.705">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at createContactTests.createContactTest.createContTest(createContactTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createContTest -->
</testsuite> <!-- Test -->
